libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Storage/KeyValueEncryptedFileStore.swift

```diff
@@ -61,7 +61,7 @@ public class KeyValueEncryptedFileStore: NSObject {
         self.encryptionKey = encryptionKey
     }
 
-    // MARK: Store management
+    // MARK: - Store management
```

Comment:
What does the dash change?

Reply:
It adds line separator.
![line-separator](https://user-images.githubusercontent.com/1095516/112241324-4417c400-8c07-11eb-8d63-8e98130be586.png)